src/NuGet/Microsoft.CodeAnalysis.Common.nuspec

```diff
@@ -21,44 +21,44 @@ Supported Platforms:
-            <dependency id="System.Threading" version="4.0.11-rc2-24027" exclude="Compile" />
-            <dependency id="System.Threading.Tasks" version="4.0.11-rc2-24027" />
-            <dependency id="System.Threading.Tasks.Parallel" version="4.0.1-rc2-24027" />
-            <dependency id="System.Threading.Thread" version="4.0.0-rc2-24027" exclude="Compile" />
-            <dependency id="System.Xml.ReaderWriter" version="4.0.11-rc2-24027" />
-            <dependency id="System.Xml.XDocument" version="4.0.11-rc2-24027" />
-            <dependency id="System.Xml.XmlDocument" version="4.0.1-rc2-24027" exclude="Compile" />
-            <dependency id="System.Xml.XPath.XDocument" version="4.0.1-rc2-24027" exclude="Compile" />
+            <dependency id="System.AppContext" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Collections" version="4.0.11-rc3-24126-00" />
+            <dependency id="System.Collections.Concurrent" version="4.0.12-rc3-24126-00" />
+            <dependency id="System.Collections.Immutable" version="1.2.0-rc3-24126-00" />
+            <dependency id="System.Console" version="4.0.0-rc3-24126-00" />
+            <dependency id="System.Diagnostics.Debug" version="4.0.11-rc3-24126-00" />
+            <dependency id="System.Diagnostics.FileVersionInfo" version="4.0.0-rc3-24126-00" exclude="Compile" />
+            <dependency id="System.Diagnostics.StackTrace" version="4.0.1-rc3-24126-00" exclude="Compile" />
+            <dependency id="System.Diagnostics.Tools" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Dynamic.Runtime" version="4.0.11-rc3-24126-00" />
+            <dependency id="System.Globalization" version="4.0.11-rc3-24126-00" />
+            <dependency id="System.IO.FileSystem" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.IO.FileSystem.Primitives" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Linq" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Linq.Expressions" version="4.0.11-rc3-24126-00" />
+            <dependency id="System.Reflection" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Reflection.Metadata" version="1.3.0-rc3-24126-00" />
+            <dependency id="System.Reflection.Primitives" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Resources.ResourceManager" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Runtime" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Runtime.Handles" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Runtime.InteropServices" version="4.1.0-rc3-24126-00" />
+            <dependency id="System.Runtime.Numerics" version="4.0.1-rc3-24126-00" />
+            <dependency id="System.Security.Cryptography.Algorithms" version="4.1.0-rc3-24126-00" />
```

Comment:
Does this need to be bumped up to 4.*2*.0 to match the change in `CommonNetCoreReferences/project.json`? If yes, please check this for all nuspecs.